README.md

```diff
@@ -14,12 +14,25 @@ might help with the notation for mathematical expressions.
 $T(n) \in O(f(n)) \iff \exists c, n_0: T(n) \leq c \cdot f(n) \forall n \geq n_0$
 
 ## Proof:
-Defining Big O: Big O notation $O(f(n))$ defines an upper bound of $f(n)$, meaning $f(n) ≤ c * g(n)$ for some constant $c > 0$ and a sufficiently large $n$, n ≥ n<sub>0</sub>
-Proving O(log<sub>2</sub>n) = O(log<sub>5</sub>n): <br />
-Recalling Change of Base Formula: log<sub>b</sub>a = $\frac{log_c a}{log<sub>c</sub>b}$ where a > 0, b > 1, c > 1 <br />
-Applying Change of Base Formula: log<sub>5</sub>n = $\frac{log<sub>2</sub>n}{log<sub>2</sub>5}$ <br />
-Setting Constants: Let k = log<sub>2</sub>5, which is a constant because both 2 and 5 are constants <br />
-Equivalent expression for log<sub>5</sub>n: log<sub>5</sub>n = $\frac{1}{k}$ log<sub>2</sub>n  $\frac{1}{k}$ is a constant multiplier. <br />
-Using Big O Defintion: By definition, for f(n) = log<sub>2</sub>n to be in O(log<sub>5</sub>n), there must exist constants c > 0 and n<sub>0</sub> > 0 such that 0 ≤ log<sub>2</sub>n ≤ c * log<sub>5</sub>n for all n ≥ n<sub>0</sub> <br />
-Substituting log<sub>5</sub>n with its relation: 0 ≤ log<sub>2</sub>n ≤ c * $\frac{1}{k}$ log<sub>2</sub>n <br />
-Choose c = k: Ensuring the inequality holds as: 0 ≤ log<sub>2</sub>n ≤ k * $\frac{1}{k}$ log<sub>2</sub>n = log<sub>2</sub>n <br />
+**Defining Big O**: Big O notation $O(f(n))$ defines an upper bound of $f(n)$, meaning $f(n) ≤ c * g(n)$ for some constant $c > 0$ and a sufficiently large $n$, n ≥ n<sub>0</sub> <br />
+### Proving O(log<sub>2</sub>n) = O(log<sub>5</sub>n): <br />
+
+**Recalling Change of Base Formula for Logarithms**: $$\log_b a = \frac{\log_c a}{\log_c b}$$ where \(a > 0\), \(b > 1\), and \(c > 1\). <br />
+
+**Applying Change of Base Formula to Relate $(\log_{2} n)$ and $(\log_{5} n)$**:
+
+$$\log_5 n = \frac{\log_2 n}{\log_2 5}$$
+
+**Setting Constants**: Let k = log<sub>2</sub>5, which is a constant because both 2 and 5 are constants <br />
+
+**Equivalent expression for (log<sub>5</sub>n)**:
+
+$$\log_5 n = \frac{1}{k} \(log_2 n)$$
+
+where $\frac{1}{k}$ is a constant multiplier.
+
+**Using Big O Defintion to Conclude Asymptotic Equivalence**: Given that $(\log_{5} n)$ is a constant multiple of $(\log_{2} n)$ , according to the definition of Big O Notation, functions are equivalent if they differ only by a constant factor. Therefore, the growth rates of $(\log_{2} n)$ and $(\log_{5} n)$ are considered equivalent in Big O Notation. <br />
```

Comment:
You need to show that they are equivalent if they differ only by a constant factor.